.github/workflows/new-release.yml

```diff
@@ -0,0 +1,66 @@
+name: New Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        type: string
+        required: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+        cache: 'yarn'
+    - name: Get version # TODO(eeeqeee) add option to increment from current version
+      id: get-version
+      run: |
+        echo "version=${{ inputs.version }}" >> $GITHUB_OUTPUT
+    - name: Set git config
+      run: |
+        git config user.name "github-actions[bot]"
+        git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+    - name: Change version
+      run: |
+        TEMP_FILE=$(mktemp)
+        jq --arg v "${{ steps.get-version.outputs.version }}" '.version = $v' package.json > $TEMP_FILE
+        cp $TEMP_FILE package.json
+        git add .
+        git commit -m "${{ steps.get-version.outputs.version }}"
+    - name: Yarn install
+      run: |
+        yarn install --immutable --immutable-cache
+    - name: Setup .env
+      run: |
+        echo $ENVFILE > apps/frontend/.env.local
+      env:
+        ENVFILE: ${{ secrets.ENVFILE }}
+    - name: Build genshin-optimizer
+      run: |
+        NX_URL_GITHUB_GO_CURRENT_VERSION="https://github.com/${{ github.repository }}/commit/$(git rev-parse HEAD)"
```

Comment:
So does this somehow get put into `process.env.NX_URL_GITHUB_GO_CURRENT_VERSION` in our `tsx` files?

If so, doesn't that mean the footer will now link to latest commit, rather than latest patch notes?

Or is the footer change for future "deploy per-pr/nightly" action?

Reply:
This sets the env var `NX_URL_GITHUB_GO_CURRENT_VERSION` which should get picked up by `process.env`
It should link to the commit it was built with, since we already have the patch notes on the home page
If env var is unset (manual build) it should be like it was before